integration-test/test_dummy_data.py

```diff
@@ -0,0 +1,185 @@
+    obs = [
+        dstore.Observation(time=time_1, value=1111.1111, metadata=obs_metadata),
+        dstore.Observation(time=time_2, value=2222.2222, metadata=obs_metadata),
+        dstore.Observation(time=time_3, value=3333.3333, metadata=obs_metadata),
+    ]
+    obs_put_request = dstore.PutObsRequest(
+        tsobs=[dstore.TSObservations(tsid=dummy_id, obs=obs)],
+    )
+    return obs_put_request
+
+
+def test_delete_timeseries(grpc_stub, dummy_timeseries_for_delete, dummy_observations_for_delete):
+    ts_add_response = grpc_stub.AddTimeSeries(dummy_timeseries_for_delete)
+    assert str(ts_add_response) == "status: -1\n"
+
+    obs_response = grpc_stub.PutObservations(dummy_observations_for_delete)
+    assert str(obs_response) == "status: -1\n"
+
+    ts_find_request = dstore.FindTSRequest(
+        station_ids=[dummy_timeseries_for_delete.metadata.station_id],
+        param_ids=[dummy_timeseries_for_delete.metadata.param_id],
+    )
+    ts_find_response = grpc_stub.FindTimeSeries(ts_find_request)
+    assert len(ts_find_response.tseries) == 1
+    assert ts_find_response.tseries[0].id == dummy_timeseries_for_delete.id
+
+    to_time = Timestamp()
+    to_time.FromDatetime(datetime(year=1999, month=9, day=9, hour=9, minute=11, second=1, tzinfo=timezone.utc))
+    obs_get_request = dstore.GetObsRequest(
+        tsids=[dummy_timeseries_for_delete.id],
+        fromtime=dummy_observations_for_delete.tsobs[0].obs[0].time,
+        totime=to_time,
+    )
+    obs_get_response = grpc_stub.GetObservations(obs_get_request)
+    assert obs_get_response.tsobs[0].tsid == dummy_timeseries_for_delete.id
+    assert len(obs_get_response.tsobs[0].obs) == 3
+
+    ts_delete_request = dstore.DeleteTSRequest(ids=[dummy_timeseries_for_delete.id])
+    ts_delete_response = grpc_stub.DeleteTimeSeries(ts_delete_request)
+    assert str(ts_delete_response) == "status: -1\n"
+
+    ts_find_response = grpc_stub.FindTimeSeries(ts_find_request)
+    assert len(ts_find_response.tseries) == 0
+
+    obs_get_response = grpc_stub.GetObservations(obs_get_request)
+    assert obs_get_response.tsobs[0].tsid == dummy_timeseries_for_delete.id
+    assert len(obs_get_response.tsobs[0].obs) == 0
+
+
+@pytest.fixture(scope="module")
+def setup_dummy_data_for_polygon(grpc_stub):
+    dummy_id = 999999990
+    dummy_str_id = "999999990"
+    ts_metadata = dstore.TSMetadata(
+        station_id=dummy_str_id,
+        param_id=dummy_str_id,
+        pos=dstore.Point(lat=80, lon=170),
+        other1="test_value1",
+        other2="test_value2",
+        other3="test_value3",
+    )
+    ts_add_request_1 = dstore.AddTSRequest(
+        id=dummy_id,
+        metadata=ts_metadata,
+    )
+    grpc_stub.AddTimeSeries(ts_add_request_1)
+
+    dummy_id = 999999991
+    dummy_str_id = "999999991"
+    ts_metadata = dstore.TSMetadata(
+        station_id=dummy_str_id,
+        param_id=dummy_str_id,
+        pos=dstore.Point(lat=90, lon=180),
```

Comment:
Although an interesting case to break the libary, I would move these points away from the north pole and the dateline.